lib: at_cmd_parser: Unsigned int and unsigned short support (#4080)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-latest.rst

    * Updated to prevent reinitialization of param list in :c:func:`modem_info_init`.

* :ref:`at_cmd_parser_readme`:

>     * Added support to parse unsigned int and unsigned short type of parameter.

Reviewer: ```suggestion
    * Added support for parsing parameters of type unsigned int or unsigned short.
```

Author: Updated
